lib/IRGen/GenReflection.cpp

```diff
@@ -1630,26 +1630,33 @@ emitAssociatedTypeMetadataRecord(const RootProtocolConformance *conformance) {
   builder.emit();
 }
 
-void IRGenModule::emitBuiltinReflectionMetadata() {
-  if (getSwiftModule()->isStdlibModule()) {
-    BuiltinTypes.insert(Context.TheNativeObjectType);
-    BuiltinTypes.insert(Context.getAnyObjectType());
-    BuiltinTypes.insert(Context.TheBridgeObjectType);
-    BuiltinTypes.insert(Context.TheRawPointerType);
-    BuiltinTypes.insert(Context.TheUnsafeValueBufferType);
+llvm::ArrayRef<CanType> IRGenModule::getSpecialBuiltinTypes() {
+  if (SpecialBuiltinTypes.empty()) {
```

Comment:
```suggestion
  if (!SpecialBuiltinTypes.empty())
    return SpecialBuiltinTypes;
```

Reply:
In this case I prefer not doing the early return as we'll need to `return SpecialBuiltinTypes` at the end of the function anyway